cluster/src/main/java/com/hartwig/pipeline/datatypes/DataType.java

```diff
@@ -56,6 +56,9 @@ public enum DataType {
     CUPPA_SUMMARY_CHART,
     CUPPA_FEATURE_PLOT,
     CUPPA_REPORT,
+    CUPPA_VIS_DATA,
+    CUPPA_VIS_PLOT,
+    CUPPA_PRED_SUMM,
```

Comment:
The CUPPA_SUMMARY_CHART, CUPPA_FEATURE_PLOT, and CUPPA_REPORT are variables for old CUPPA and can be removed. These variables no longer appear anywhere in pipeline5, and are also no longer used by ORANGE